splunk/provider.go

```diff
@@ -90,6 +91,7 @@ func providerResources() map[string]*schema.Resource {
 		"splunk_saved_searches":              savedSearches(),
 		"splunk_indexes":                     index(),
 		"splunk_configs_conf":                configsConf(),
+		"splunk_dashboards":                  splunkDashboards(),
```

Comment:
@ani0989 
Can we change the resource name to `splunk_data_ui_views`? 
Since that is how all the resources are named and we can continue the trend of `endpoint` prefixed with `splunk`.